source/B2CWebApi/Models/RequestAggregatedMeasureDataMarketDocumentV1.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Energinet.DataHub.EDI.B2CWebApi.Models;
+
+public record RequestAggregatedMeasureDataMarketDocumentV1(
+    string BusinessReason,
+    RequestAggregatedMeasureDataSeriesV1 Series);
+
+public record RequestAggregatedMeasureDataSeriesV1(
+    string? MarketEvaluationPointType,
+    string? MarketEvaluationSettlementMethod,
+    string StartDateAndOrTimeDateTime,
+    string? EndDateAndOrTimeDateTime,
```

Comment:
```suggestion
    string EndDateAndOrTimeDateTime,
```
The BFF will always fill this out?

Reply:
agreed!